src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -132,7 +132,7 @@ private static Signers SignersFromJson(JArray _params)
         [RpcMethod]
         protected virtual JObject InvokeFunction(JArray _params)
         {
-            UInt160 script_hash = UInt160.Parse(_params[0].AsString());
+            UInt160 script_hash = UInt160.Parse(ToScriptHash(_params[0].AsString()));
```

Comment:
This is not #355, it's something different and probably not needed. #355 is about `getcontractstate`.

Reply:
ok